src/node/node_state.h

```diff
@@ -198,7 +160,7 @@ namespace ccf
 #endif
     }
 
-    void initialise_startup_snapshot(bool recovery = false)
+    bool initialise_startup_snapshot(bool recovery = false)
```

Comment:
May be worth documenting what the boolean is, it's no intuitive that true indicates ledger verification is needed but false doesn't. An enum might be a helpful readability investment here.

Reply:
A lot of this code will be removed after 2.x so I've only added a comment to detail the `bool` here.